readme.txt

```diff
@@ -190,6 +190,10 @@ This version adds an option to refresh the malware scan results on demand, as we
 
 == Changelog ==
 
+= 1.8.23 =
+* Add Automatic Secret Keys Updater
+* Modify sucuri-scanner.pot to be updated via wp-cli i18n make-pot
```

Comment:
"Modify sucuri-scanner.pot to be updated via wp-cli i18n make-pot" is not a change that affects users in any way, so I think it's not useful to include it